Packs/ZeroFox/Integrations/ZeroFox/README.md

```diff
@@ -4,18 +4,18 @@ This integration was integrated and tested with versions 1.0 and 2.0 of ZeroFox.
 ## Configure ZeroFox in Cortex
 
 
-| **Parameter** | **Required** |
-| --- | --- |
-| URL (e.g., https://api.zerofox.com/) | True |
-| Username | True |
-| Password | True |
-| Fetch only escalated alerts | False |
-| Trust any certificate (not secure) | False |
-| Use system proxy settings | False |
-| First fetch timestamp (&lt;number&gt; &lt;time unit&gt;, e.g., 12 hours, 7 days) | False |
-| Fetch Limit | True |
-| Fetch incidents | False |
-| Incident type | False |
+    | **Parameter** | **Required** |
+    | --- | --- |
+    | URL (e.g., <https://api.zerofox.com/>) | True |
+    | Username | True |
+    | Password | True |
+    | Fetch only escalated alerts | False |
+    | Trust any certificate (not secure) | False |
+    | Use system proxy settings | False |
+    | First fetch timestamp (&lt;number&gt; &lt;time unit&gt;, e.g., 12 hours, 7 days) | False |
+    | Fetch Limit | True |
+    | Fetch incidents | False |
+    | Incident type | False |
```

Comment:
```suggestion
| **Parameter** | **Required** |
| --- | --- |
| URL (e.g., <https://api.zerofox.com/>) | True |
| Username | True |
| Password | True |
| Fetch only escalated alerts | False |
| Trust any certificate (not secure) | False |
| Use system proxy settings | False |
| First fetch timestamp (&lt;number&gt; &lt;time unit&gt;, e.g., 12 hours, 7 days) | False |
| Fetch Limit | True |
| Fetch incidents | False |
| Incident type | False |
```
Is there a specific reason for indenting the table? Typically, tables in Markdown don't need indentation, and it might affect their formatting and appearance.

Reply:
it probably was an automatic update by my linter, I'll reverse that

Reply:
@YairGlik  done!